tsl/test/sql/cagg_joins.sql.in

```diff
@@ -53,10 +53,12 @@ SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    name
 FROM conditions, devices
 WHERE conditions.device_id = devices.device_id
-GROUP BY name, bucket;
+GROUP BY name, 1;
```

Comment:
Nit: Why is this change needed?

Reply:
just to be consisted, in most cases the param number is listed hence this change.